src/index.js

```diff
@@ -89,7 +92,11 @@ export default class VueRouter {
     }
 
     history.listen(route => {
-      this.app._route = route
+      this.apps.forEach((app) => {
+        if (app) {
```

Comment:
Thats what I thought as well, but the syntax check wouldn't let it pass. Don't remember exactly, but it was along the lines of "Cannot set property on a possibly null value".